src/app/page.tsx

```
@@ -120,57 +208,187 @@ export default function Home() {
-            >
-              Download JSON
-            </button>
-          </div>
-          <pre className="bg-gray-100 p-2 rounded overflow-x-auto text-xs text-gray-900">
-            {JSON.stringify(result, null, 2)}
-          </pre>
-        </div>
-      )}
+
+      {result &&
+        result.reviews &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)